jsk_mbzirc_common/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(jsk_mbzirc_common)
-
+set(CMAKE_BUILD_TYPE Release)
```

Comment:
Sorry for that,
I use qt creator to test if I could debug... 
next time when I commit I will remove this